lib/src/common/model/repos_commits.dart

```diff
@@ -123,3 +123,65 @@ class CommitFile {
       ..json = input;
   }
 }
+
+/// Model class for a commit comment.
+///
+/// See https://developer.github.com/v3/repos/comments
+class CommitComment {
+  /// Id of the comment
+  final int id;
+
+  /// Relative path of the file on which the comment has been posted
+  final String path;
+
+  /// Line on file
+  final int line;
+
+  /// Position on the diff
+  final int position;
+
+  /// SHA of the commit where the comment has been made
+  final String commitId;
+
+  final DateTime createdAt;
+
+  /// Can be equals to [createdAt]
+  final DateTime updatedAt;
+
+  /// https://github.com/...
```

Comment:
Should this be a full URL ?

Reply:
I actually wanted to show the pattern of the returned URL, but I can remove it if your prefer

Reply:
oh, gotcha. that makes sense.